### PR TITLE
Homepage layout proposal

### PR DESCRIPTION
**Pull request by muczc1wek**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,12 +2,45 @@
...
+
+
+
+-  :fontawesome-brands-github:{ .lg .middle } __Repository__
+
+    ---
+
+    This site has a public repo hosted on GitHub.
+
+    [:octicons-arrow-right-24: GitHub](https://github.com/Silver-Ore-Team/zBassMusic)
```
Comment (piotrmacha) [inline docs/index.md:43]: Change the repo link to GMC

**Review by piotrmacha — CHANGES_REQUESTED** [review]

**Comment by muczc1wek** [comment]
obraz

**Comment by muczc1wek** [comment]
I think the PR is ready, I would also provide polish translation, just let me know if the layout is good enough

File: docs/zengin/index.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+    Create new animations and put them into the game.
+
+    [:octicons-arrow-right-24: Animations](anims/index.md)
+
+
+-  :material-code-block-tags:{ .lg .middle } __Union SDK__
+
+    ---
+
+    Learn how to create plugins with Union SDK, a community software development kit for ZenGin.
```
Comment (kamilkrzyskow) [inline docs/zengin/index.md:38]: community-made would be more correct, there is no pro version of union

**Review by kamilkrzyskow — CHANGES_REQUESTED** [review]

**Comment by kamilkrzyskow** [comment]
Additionally, the position of the links isn't synced to the bottom, I once helped out with this via extra CSS, should be a small fix 🤔 
Will edit the comment in a bit when I find it....

**Comment by kamilkrzyskow** [comment]
Otherwise the layout and content seems fine, you can start translating imo 👌

File: docs/index.md [hunk 3]
```diff
@@ -2,12 +2,45 @@
...
+    [:octicons-arrow-right-24: Discord](https://discord.gg/mCpS5b5SUY)
+
+
+-  :octicons-git-pull-request-16:{ .lg .middle } __Contribution__
+
+    ---
+
+    Feel free to open a pull request with your article or propose changes. Here is detailed contribution guide.
+
+    [:octicons-arrow-right-24: How to Contibute](./contribute/index.md)
```
Comment (kamilkrzyskow) [inline docs/index.md:25]: The `./` is still here

**Review by kamilkrzyskow — CHANGES_REQUESTED** [review]

**Comment by muczc1wek** [comment]
@kamilkrzyskow  Is that what you wanted to achieve?
obraz

**Comment by kamilkrzyskow** [comment]
Yes, the links are on the bottom even if the content isn't as big as in the other column. 
Everything is "unified" for lack of a better word.

File: docs/index.md (hunk 3 above)
Comment (muczc1wek) [reply docs/index.md:25]: Yeah I accidentally deleted it in zengin/index not index 😄

**Comment by muczc1wek** [comment]
I am currently connected with my PC by remote desktop so the graphics is a bit deformed

**Comment by kamilkrzyskow** [comment]
The changes LGTM, good CSS placement 👍 , the `outline` comment didn't have to be there though.
I forgot to remove it from the answer, this was for me to figure this solution out 😅

**Comment by kamilkrzyskow** [comment]
Please also add this

https://github.com/Gothic-Modding-Community/gmc/assets/34622465/d3996056-c092-41d0-97aa-2acda821230a

```css
.grid.cards li > p:first-child > strong {
  vertical-align: middle;
}
```

File: docs/zengin/index.pl.md [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+    [:octicons-arrow-right-24: Animations](anims/index.md)
+
+
+-  :material-code-block-tags:{ .lg .middle } __Union SDK__
+
+    ---
+
+    Naucz się jak tworzyć pluginy za pomocą Union SDK, stworzonego przez fanów narzędzia deweloperskiego dla silnika ZenGin.
+
+    [:octicons-arrow-right-24: Getting Started](union/sdk/getting_started.md)
```
Comment (kamilkrzyskow) [inline docs/zengin/index.pl.md:40]: Link names should be localized even though the pages aren't localized yet.

**Review by kamilkrzyskow — CHANGES_REQUESTED** [review]

File: docs/zengin/index.pl.md (hunk 4 above)
Comment (muczc1wek) [reply docs/zengin/index.pl.md:40]: Yeah, i noticed that too

File: docs/zengin/index.pl.md [hunk 5]
```diff
@@ -1,3 +1,41 @@
...
+
+<div class="grid cards" markdown>
+
+-  :material-code-braces:{ .lg .middle } __Skrypty Gothica__
+
+    ---
+
+    ZenGin używa własnego języka skryptowego o nazwie Daedalus. Sprawdź jego funkcje, klasy i extendery.
+
+    [:octicons-arrow-right-24: Scripts](scripts/index.md)
```
Comment (kamilkrzyskow) [inline docs/zengin/index.pl.md:13]: English name

**Review by kamilkrzyskow — CHANGES_REQUESTED** [review]

File: docs/zengin/index.pl.md (hunk 5 above)
Comment (muczc1wek) [reply docs/zengin/index.pl.md:13]: I will fix that later, I need some rest from pc

**Review by kamilkrzyskow — APPROVED** [review]

**Pull request merged**